config-template.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copy this file to config.js and fill it in with your own information.
+ */
+module.exports = {
```

Comment:
I mean, sure, if people know how to do that. Maybe I'm missing it but in the step-by-step guide we're making to have people do this, the only thing I find is : Don't forget to fill _config.js_ with your information!

How does a user know what information they need/where to get it? Am I missing that?